docs/ssr.mdx

```diff
@@ -41,6 +41,47 @@ import { renderToString } from 'react-dom/server'
 import createEmotionServer from '@emotion/server/create-instance'
 import createCache from '@emotion/cache'
 
+const key = 'custom'
+const cache = createCache({ key })
+const { extractCritical2 } = createEmotionServer(cache)
+
+let element = (
+  <CacheProvider value={cache}>
+    <App />
+  </CacheProvider>
+)
+
+let { html, css, globalCss, ids } = extractCritical2(renderToString(element))
+
+res
+  .status(200)
+  .header('Content-Type', 'text/html')
+  .send(`<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My site</title>
+    <style data-emotion="global-css">${globalCss.reduce((acc, curr) => acc + curr)}</style>
+    <style data-emotion="${key} ${ids.join(' ')}">${css}</style>
```

Comment:
I believe this should be enough...

Reply:
Added `constructStyleTags`, but after the refactoring, it turned out to be trivial to add these (as all styles are an array, where the global styles are added first, and the rest of the regular styles last).. Let me know if you still think it's useful to have it.